js/app.js

```diff
@@ -1,41 +1,49 @@
-// --- API KEY: AIzaSyCKNNUXyaTDWJp14VhcwraU-mY7o4A27HY --- //
+// Global Variables
+var tubeKey = 'AIzaSyCKNNUXyaTDWJp14VhcwraU-mY7o4A27HY';
+var tubeUrl = 'https://www.googleapis.com/youtube/v3/search';
 
-// --- The endpoint: "https://www.googleapis.com/youtube/v3/search" --- //
-
-$(function(){
-  $('#search-button').on("click", function(event){
-    var searchTerm = $('#query').val();
-    $('#top_movies .clearfix h2').text(searchTerm);
-    getRequest(searchTerm);
-  });
+// --- Function takes query term, displays it and calls getRequest function with query -- //
+$(function() 
+{
+	$('#search-button').on("click", function(event)
```

Comment:
Try not to mix quotes. If your using single, use single everywhere. It looks neater.